docs/userGuide/syntax/boxes.mbdf

```diff
@@ -169,6 +200,7 @@ border-left-color | `String` | `null` | Overrides border-color for the left bord
 color | `String` | `null` | Color of the text.
 dismissible | `Boolean` | `false` | Adds a button to close the box to the top right corner.
 icon | `String` | `null` | Inline MarkDown text of the icon displayed on the left.
+icon-size | `String` | `null` | Resizes the icon. Supports integer-scaling of the icon dimensions e.g. `2x`, `3x`, `4x`, etc.  
```

Comment:
Would `1x` be a better default?

Reply:
I am not sure that would be better. If `1x` were documented as the default, it could confuse the user to interpret it as though the class `fa-1x` would be set for the icon by default - which is not the case. 
Setting the default as `1x` in the `TipBox.vue` component would be redundant as well because by default the icon's size is always `1x`.